src/ts/Voice.ts

```diff
@@ -105,8 +105,8 @@ export default class Voice
 
                     // Parse installed voice packages
                     Neutralino.filesystem.readDirectory(voiceDir)
-                        .then(async (files) => {
-                            files = files.filter((file) => file.type == 'DIRECTORY')
+                        .then(async (filesString) => {
+                            const files = filesString.filter((file) => file.type == 'DIRECTORY')
```

Comment:
Why are you changing all "files" variables to "filesString" when there is no real need as you just add overhead when then proceeding to make a new const for "files"